src/components/app.jsx

```diff
@@ -80,23 +70,49 @@ const App = React.createClass({
                 <div className="col-md-9">
                   <p id="baseline">
                     <span className="message">
-                      <FormattedMessage {...messages.header} />
+                      <FormattedMessage id="header"/>
+                      <small>
+                        <ExternalLink href = "https://www.openfisca.fr" target = "_blank">
```

Comment:
No space around `=` on attributes. This is not an imperative allocation, but an XML-style declaration. The fact that JSX supports spaces should not make it harder for a human eye used to XML to read this code.
Please, this is repeated on several occurrences and was already requested for change in the previous review.